src/pages/study-room/hooks/usePopover.ts

```diff
@@ -0,0 +1,27 @@
+import { useState } from 'react';
+
+export interface UsePopover {
+  anchorEl: HTMLButtonElement | null;
+  onOpen: () => void;
+  onClose: () => void;
+}
+
+const usePopover = () => {
+  const [anchorEl, setAnchorEl] = useState<HTMLButtonElement | null>(null);
```

Comment:
이렇게 타입을 재활용해줄 수 있을 것 같아요~
```suggestion
  const [anchorEl, setAnchorEl] = useState<UsePopover['anchorEl']>(null);
```